lib/code_corps/policy/user_skill.ex

```diff
@@ -1,11 +1,9 @@
 defmodule CodeCorps.Policy.UserSkill do
-  alias CodeCorps.UserSkill
-  alias CodeCorps.User
-  alias Ecto.Changeset
+  alias CodeCorps.{UserSkill, User}
 
-  def create?(%User{admin: true}, %Changeset{}), do: true
-  def create?(%User{id: id}, %Changeset{changes: %{user_id: user_id}}), do: id == user_id
-  def create?(%User{}, %Changeset{}), do: false
+  def create?(%User{admin: true}, %{}), do: true
+  def create?(%User{id: id}, %{user_id: user_id}), do: id == user_id
```

Comment:
@snewcomer The params we send here use strings as keys. Additionally, there's an issue with id values in the params map all being strings (this is what ember sends), so to fix the problem here you need to 

- replace this line with `def create?(%User{id: id}, %{"user_id" => user_id}), do: id == user_id`
- update the policy test to match
- add `plug CodeCorpsWeb.Plug.IdsToIntegers` into the controller, right after the `DataToAttributes` plug.

Reply:
@begedin thats a great catch.  Fixed.